Makefile

```
@@ -25,29 +26,35 @@ FPMCOMMON= \
 release: release-x64 release-mac
 
 release-bin:
-	docker run -v `pwd`:$(SOURCE_PATH) -t --env GOOS=linux --env GOARCH=amd64  -i gor go build -o gor -tags netgo $(LDFLAGS)
+	docker run -v `pwd`:$(SOURCE_PATH) -t --env GOOS=linux --env GOARCH=amd64  -i $(CONTAINER) go build -o $(BIN_NAME) -tags netgo $(LDFLAGS)
+
+release-bin-mac:
+	go build -o $(BIN_NAME) $(MAC_LDFLAGS)
```

It might be less problematic to explicitly provide `GOOS` env... like
```suggestion
	GOOS=darwin go build -o $(BIN_NAME) $(MAC_LDFLAGS)
```

---

Fixed.